bazel/repository_locations.bzl

```diff
@@ -248,4 +248,14 @@ REPOSITORY_LOCATIONS = dict(
         sha256 = "fcdebf54c89d839ffa7eefae166c8e4b551c765559db13ff15bff98047f344fb",
         urls = ["https://storage.googleapis.com/quiche-envoy-integration/2a930469533c3b541443488a629fe25cd8ff53d0.tar.gz"],
     ),
+    com_google_cel_cpp = dict(
+        sha256 = "a0e6a6ccf25c1e57ba3c7a997edd9279f8e0b0112e0a3ac705a5568fa32792fc",
+        strip_prefix = "cel-cpp-d56f26adb53d0f41508a909d15e74d9ffb0e8a6c",
+        urls = ["https://github.com/google/cel-cpp/archive/d56f26adb53d0f41508a909d15e74d9ffb0e8a6c.tar.gz"],
+    ),
+    com_google_re2 = dict(
```

Comment:
use the canonical workspace name `com_googlesource_code_re2` defined in https://github.com/google/re2/blob/master/WORKSPACE

Reply:
Ugh, it's brought in externally. Let me try to fix the workspace in CEL.

Reply:
Fixed.